components/apps/ddm/forms/FormAddReplication.tsx

```diff
@@ -45,6 +45,10 @@ export default function FormAddReplication(props: {
     }
   }
 
+  function formValid() {
+    return !!providerID;
```

Comment:
btw you can also do `Boolean(providerID)`, I find it a bit cleaner but personal preference